### PR TITLE
[core] Disclaimer functionality proposal

### PR DESCRIPTION
**Pull request by ridz1208**
This PR serves only as an example of the behaviour expected for this feature

### CODE REVIEW

File: modules/new_profile/php/NDB_Form_new_profile.class.inc [hunk 1]
```diff
@@ -210,5 +213,12 @@ class NDB_Form_new_profile extends NDB_Form
         return $errors;
     }
 
+    function getDisclaimer($module) {
+
+        $db =& Database::singleton();
+
+        return $db->pselectOne("SELECT Value FROM disclaimer WHERE module=:mod", array("mod"=>$module));
```
Comment (xlecours) [inline modules/new_profile/php/NDB_Form_new_profile.class.inc]: phpcs?

File: SQL/Archive/17.1/2017-03-15-Disclaimer.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+CREATE TABLE `disclaimer` (`module` varchar(100) NOT NULL, `value` TEXT DEFAULT NULL);
```
Comment (xlecours) [inline SQL/Archive/17.1/2017-03-15-Disclaimer.sql]: What about module_name and displayed_string for column names?

Comment (xlecours) [inline SQL/Archive/17.1/2017-03-15-Disclaimer.sql]: If we use pselectOne are we assuming there can be only 1 disclaimer per module? If yes, a unique/primary key would be awesome

**Comment by ridz1208** [comment]
@xlecours this is a very very badly implemented 5-min version of what I would like to see. Only to be demoed on tuesday and then deleted, burned, buried and forever forgotten in the abyss of time itself.

**Comment by ridz1208** [comment]
will be discussed on Tuesday, afterwhich this PR will be deleted

**Pull request closed without merging**